src/elements/radio-button/radio-button-panel/radio-button-panel.ts

```diff
@@ -50,8 +51,10 @@ class SbbRadioButtonPanelElement extends SbbPanelMixin(
 
   /** Size variant. */
   @property({ reflect: true })
-  @getOverride((i, v) => (i.group?.size ? (i.group.size === 'xs' ? 's' : i.group.size) : v))
-  public accessor size: SbbPanelSize = 'm';
+  @getOverride((i, v) =>
+    i.group?.size ? (isLean() || i.group.size === 'xs' ? 's' : i.group.size) : v,
```

Comment:
with lean, the standard group size should already be xs?
```suggestion
    i.group?.size ? (i.group.size === 'xs' ? 's' : i.group.size) : v,
```